make/CompileJavaModules.gmk

```diff
@@ -117,6 +116,50 @@ $(eval $(call SetupJavaCompilation, $(MODULE), \
 
 TARGETS += $($(MODULE))
 
+################################################################################
+# Setup compilation for value classes in the module
+# TBD: When $(DOCLINT) was included there was an NPE in JavacTypes.getOverriddenMethods
+
+# Directory and file name suffix for jar file containing value classes
+VALUECLASSES_STR := valueclasses
+
+ifneq ($(COMPILER), bootjdk)
+    MODULE_VALUECLASS_SRC_DIRS := $(call FindModuleValueClassSrcDirs, $(MODULE))
```

Comment:
Please use 2 spaces for logic block indentation. (see https://openjdk.org/groups/build/doc/code-conventions.html)